bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```diff
@@ -20,14 +20,14 @@ export default function userLeaving(credentials, userId, connectionId) {
     userId,
   };
 
-  const User = Users.findOne(selector);
+  const user = Users.findOne(selector);
```

Comment:
We typically name the result of Collection.find*(...) with capital letter - `User` in this case. I'd recommend leaving it as before